src/TLuaInterpreter.cpp

```diff
@@ -2281,7 +2281,30 @@ int TLuaInterpreter::saveProfile(lua_State* L)
     }
 }
 
-// openUserWindow( session, string window_name )
+int TLuaInterpreter::setUserWindowFontSize(lua_State* L)
+{
+    QString windowName = "";
+    int size = 10; // is there a better default value for this?
```

Comment:
I think a reasonable default would be the default of the main window the user has?